packages/website/src/components/editor/config.ts

```diff
@@ -2,18 +2,38 @@ import type Monaco from 'monaco-editor';
 import type { JSONSchema4 } from '@typescript-eslint/utils/dist/json-schema';
 import { getTypescriptOptions } from '../config/utils';
 
+type StringMap = Map<string, string>;
+declare module 'typescript' {
+  /**
+   * Map of available libraries
+   *
+   * The key is the key used in compilerOptions.lib
+   * The value is the file name
+   */
+  const libMap: StringMap;
+}
```

Comment:
can we please put this in `typings/typescript.d.ts`?

It's nice to make it clear what "private" surface we expose.